app/assets/javascripts/discourse/components/choose-message.js.es6

```diff
@@ -0,0 +1,65 @@
+import debounce from "discourse/lib/debounce";
+import { searchForTerm } from "discourse/lib/search";
+import { observes } from "ember-addons/ember-computed-decorators";
+
+export default Ember.Component.extend({
+  loading: null,
+  noResults: null,
+  messages: null,
+
+  @observes("messageTitle")
+  messageTitleChanged() {
+    this.setProperties({
+      loading: true,
+      noResults: true,
+      selectedTopicId: null
+    });
+    this.search(this.get("messageTitle"));
+  },
+
+  @observes("messages")
+  messagesChanged() {
+    const messages = this.get("messages");
+    if (messages) {
+      this.set("noResults", messages.length === 0);
+    }
+    this.set("loading", false);
+  },
+
+  search: debounce(function(title) {
+    const self = this,
+      currentTopicId = this.get("currentTopicId");
+
+    if (Em.isEmpty(title)) {
+      self.setProperties({ messages: null, loading: false });
+      return;
+    }
+
+    searchForTerm(title, {
+      typeFilter: "private_messages",
+      searchForId: true
+    }).then(function(results) {
+      if (results && results.posts && results.posts.length > 0) {
+        self.set(
+          "messages",
+          results.posts
+            .mapBy("topic")
+            .filter(t => t.get("id") !== currentTopicId)
+        );
+      } else {
+        self.setProperties({ messages: null, loading: false });
+      }
+    });
+  }, 300),
+
+  actions: {
+    chooseMessage(message) {
+      const messageId = Em.get(message, "id");
+      this.set("selectedTopicId", messageId);
+      Ember.run.next(() =>
```

Comment:
Could this be an `afterRender` instead?

Reply:
I tried replacing it with

```
Ember.run.scheduleOnce("afterRender", () => {
  $(`#choose-message-${messageId}`).prop("checked", "true")
});
```

but it's not working in this case, the topic is not getting selected.